index.js

```diff
@@ -133,25 +140,10 @@ Object.assign(updater, {
   },
 
   updateTaskDefinitionImage(taskDefinition, containerNames, image) {
-    if (!_.isArray(containerNames)) containerNames = [containerNames];
-
-    var newTaskDefinition = _.clone(taskDefinition);
-    containerNames.forEach((containerName) => {
-      var containerIndex = _.findIndex(newTaskDefinition.containerDefinitions, (containerDefinition) => {
-        return containerDefinition.name === containerName;
-      });
-      
-      newTaskDefinition.containerDefinitions[containerIndex].image = image;
-    });
-
-    return _.pick(newTaskDefinition, [
-      'containerDefinitions',
-      'family',
-      'networkMode',
-      'placementConstraints',
-      'taskRoleArn',
-      'volumes'
-    ]);
+    return taskDefinition.containerDefinitions.map(
+      def =>
+        [].concat(containerNames).includes(def.name) ? { ...def, image } : def
```

Comment:
`[].concat(containerNames)` array-ifies the arguments:
```javascript
[].concat('boop')   // => ['boop']
[].concat(['boop'])   // => ['boop']
```

Reply:
that's a nice hack